Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -67,6 +67,11 @@ namespace ROS2
             AZStd::lock_guard<AZStd::mutex> lck(m_statusLock);
             m_status.clear();
         }
+
+         // Begin an undo batch for prefab creation process
+        AzToolsFramework::UndoSystem::URSequencePoint* currentUndoBatch;
+        AzToolsFramework::ToolsApplicationRequests::Bus::BroadcastResult(currentUndoBatch, &AzToolsFramework::ToolsApplicationRequests::BeginUndoBatch, "Robot Importer prefab creation");
```

Comment:
There is an early return between this call and the End call, such as lline 76/81. Would that persist and lead to a weird behavior?

Reply:
Definitely it should finish batch on early return.

Reply:
It's an oversight on my end. I'll fix it.